kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -1078,6 +1162,7 @@ class SchemaUtilsSuite extends AnyFunSuite {
       validateUpdatedSchemaAndGetUpdatedSchema(
```

Comment:
ideally these would also be a test method, but good to not do this in this PR since it's already unideal